client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -85,23 +86,59 @@ class MappedDomainType extends React.Component {
 			);
 		}
 
+		const setupInstructionsMessage = translate(
+			'Follow these instructions to set up your domain mapping.'
+		);
+
 		return (
 			<React.Fragment>
 				<div>
-					<p>{ primaryMessage }</p>
-					{ ! isSubdomain( domain.name ) && (
-						<ul className="mapped-domain-type__name-server-list">
-							{ WPCOM_DEFAULT_NAMESERVERS.map( ( nameServer ) => {
-								return <li key={ nameServer }>{ nameServer }</li>;
-							} ) }
-						</ul>
-					) }
+					<p>{ setupInstructionsMessage }</p>
+					{ this.renderRecommendedSetupMessage( primaryMessage ) }
+					{ domain.aRecordsRequiredForMapping && this.renderARecordsMappingMessage() }
 				</div>
 				<div className="mapped-domain-type__small-message">{ secondaryMessage }</div>
 			</React.Fragment>
 		);
 	}
 
+	renderRecommendedSetupMessage( primaryMessage ) {
+		const { domain, translate } = this.props;
+
+		const recommendedSetupTitle = translate( 'Recommended setup' );
+		return (
+			<FoldableFAQ id="recommended-mapping-setup" question={ recommendedSetupTitle } expanded>
```

Comment:
I feel we could also avoid that unnecessary variable as well here but maybe that's personal preference:

```suggestion
		const recommendedSetupTitle = translate( 'Recommended setup' );
		return (
			<FoldableFAQ
				id="recommended-mapping-setup"
				question={ translate( 'Recommended setup' ) }
				expanded
			>
```